content.ts

```diff
@@ -219,6 +227,33 @@ export class Content {
 
         return this;
     }
+
+    static async list(client: CortexApiClient, paginationOpts?: { cursor?: string, pageSize?: number }): Promise<ContentListRequestResult> {
```

Comment:
Can we give pagination opts an explicit type that we export? That will make it easier to go back and add docstrings on the types that show up in autocomplete/generated docs.